.gitignore

```diff
@@ -8,7 +8,6 @@ __pycache__/
 .venv/
 venv/
 node_modules/
-config.yaml
```

Comment:
I think this was in gitignore so that developers can set overrides for local testing. So this doesn't need to be removed.

Reply:
Agreed